server/sonar-server/src/main/java/org/sonar/server/es/EsUtils.java

```diff
@@ -79,4 +82,17 @@ public static String formatDateTime(@Nullable Date date) {
     }
     return null;
   }
+
+  public static void setShards(NewIndex index, Settings settings) {
```

Comment:
I don't think this should be extracted in "yet another util method". Why not moving it to NewIndex ?